com.slack.Slack.json

```diff
@@ -10,6 +10,7 @@
     "finish-args": [
         "--share=ipc",
         "--socket=x11",
+        "--socket=wayland",
```

Comment:
See existing issues and PRs, this has been discussed many times already. Wayland support is disabled by default because it is experimental and not supported upstream.